### PR TITLE
[WSTEAMA-35] Chameleon Headers and Footers

### PR DESCRIPTION
**Pull request by RetiredBlob**
Resolves https://jira.dev.bbc.co.uk/browse/WSTEAMA-35

- Update Header and Footer to use new Chameleon logos
- Updates / simplification to psammead-brand to match the design
- Fix issue on topic page where it was slightly narrower than intended on desktop

Known issues:
- Missing logos for japanese and some public services - more work to follow if/when designers provide these

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by greenc05** [comment]
I realise this is in progress, incase you aren't aware we have a couple of issues at the mo..

- **Page zoomed to 200%** - Lost the spacing around the brand logo

Screen shot from live site: 
image

Screen shot from PR: 
image

- **Font resized to largest default setting** - Same issues as page zoom

- **No CSS** - The logo is too large

Screen shot from live site:
image

Screen shot from PR:
image

File: src/app/legacy/psammead-brand/src/index.jsx [hunk 1]
```diff
@@ -43,17 +36,19 @@ const SvgWrapper = styled.div`
 
 const Banner = styled.div`
   background-color: ${props => props.backgroundColour};
-  ${({ svgHeight }) =>
-    conditionallyRenderHeight(svgHeight, PADDING_AROUND_SVG_BELOW_400PX)}
+  height: 44px;
```
Comment (greenc05) [inline src/app/legacy/psammead-brand/src/index.jsx]: Please could we use a relative value?

File: src/app/legacy/psammead-brand/src/index.jsx [hunk 2]
```diff
@@ -43,17 +36,19 @@ const SvgWrapper = styled.div`
...
   width: 100%;
   padding: 0 ${GEL_SPACING};
 
   @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
-    ${({ svgHeight }) =>
-      conditionallyRenderHeight(svgHeight, PADDING_AROUND_SVG_ABOVE_400PX)}
+    height: 56px;
```
Comment (greenc05) [inline src/app/legacy/psammead-brand/src/index.jsx]: Please could we use a relative value?

File: src/app/legacy/psammead-brand/src/index.jsx [hunk 3]
```diff
@@ -43,17 +36,19 @@ const SvgWrapper = styled.div`
...
     padding: 0 ${GEL_SPACING_DBL};
   }
 
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    height: 64px;
```
Comment (greenc05) [inline src/app/legacy/psammead-brand/src/index.jsx]: Please could we use a relative value?

**Comment by greenc05** [comment]
When you are ready these are the required a11y swarm checks:

- Meets [accessibility acceptance criteria](https://paper.dropbox.com/doc/Chameleon-Brand-logo-update-for-banner-and-footer-Accessibility-Acceptance-Criteria--BmH2qGSUyIuON2SBJoDk0QqPAg-97HH5kxP4hOyQCFxnNQfj)
- Run Axe
- All [supported AT](https://bbc.github.io/accessibility-news-and-you/assistive-technology/testing.html)

**Review by rebeccamcginn — APPROVED** [review]

**Comment by rebeccamcginn** [comment]
Chromatic isn't happy with 3 tests

**Review by amoore108 — APPROVED** [review]

**Pull request merged**